synapse/handlers/e2e_room_keys.py

```diff
@@ -377,8 +377,8 @@ async def delete_version(self, user_id: str, version: Optional[str] = None) -> N
         """Deletes a given version of the user's e2e_room_keys backup
 
         Args:
-            user_id(str): the user whose current backup version we're deleting
-            version(str): the version id of the backup being deleted
+            user_id: the user whose current backup version we're deleting
+            version: the version id of the backup being deleted
```

Comment:
What does `version=None` mean here---looks like it means "the most recent backup"?

(FWIW my guess would have been "delete all versions")

Reply:
This copies the docstring from `delete_e2e_room_keys_version`:

```suggestion
            version: Optional. the version ID of the backup version we're deleting
                If missing, we delete the current backup version info.
```